src/recordpage.cpp

```diff
@@ -664,6 +685,44 @@ void RecordPage::onOutputFileNameSelected(const QString& filePath)
     }
 }
 
+void RecordPage::onApplicationConfigAdded()
+{
+    saveApplicationConfig(QLatin1String("Config %1").arg(QString::number(ui->launchConfigComboBox->count() + 1)));
```

Comment:
QString::number isn't needed anymore now

Reply:
it is needed. int does not convert implicit to QLatin1String.

Reply:
I can change it to QString, since arg on a QString does support int